scout/server/config.py

```diff
@@ -21,4 +21,4 @@
 ACCEPT_LANGUAGES = ['en', 'sv']
 
 # FEATURE FLAGS
-SHOW_CAUSATIVES = False
+SHOW_CAUSATIVES = True
```

Comment:
Actually a mistake from testing, but yeah, I don't quite see why such a nifty feature is left disabled by default? @moonso, just because it wasn't quite ready at the time??

Reply:
This will anyhow be overridden with server specific configs, so no real harm in keeping it on here.